### ISSUE TITLE
"Output type 'Console Application' is not supported by one or more of the project's targets." for portable console applications

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#6069]
We've turned VbcCore and CscCore into console applications, but we now complain about this in portable:

```
Severity	Code	Description	Project	File	Line	Source
Error		Output type 'Console Application' is not supported by one or more of the project's targets.	VbcCore	0	IntelliSense
Error		Output type 'Console Application' is not supported by one or more of the project's targets.	CscCore	0	IntelliSense
```
Not sure if we need now to have a ConsoleApplication concept inside PCL, or whether we should just stop complaining.

**Comment by davkean** [comment]
@Pilchie I think we want to take this in VS Update 1, this is going to be annoying for all Roslyn development without it.

**Comment by dpoeschl** [comment]
@davkean @Pilchie This just looks like an abandoned milestone disagreement conversation. :smile:

**Comment by ctaggart** [comment]
Just ran into this. Does this mean it is not possible to build Roslyn master with Visual Studio 2015 RTM? Update 1 is needed?

**Comment by ctaggart** [comment]
Nevermind. I haven't built everything from the command line first as the docs say to do:
https://github.com/dotnet/roslyn/wiki/Building%20Testing%20and%20Debugging#getting-the-code

**Comment by davkean** [comment]
Got a fix for this.

**Comment by davkean** [comment]
This has been fixed internally for 1.2.
